custom_components/tuya_local/devices/luko_dehumidifier.yaml

```diff
@@ -0,0 +1,66 @@
+name: Luko Dehumidifier
+products:
+  - id: 4w2fzgr62c1oyjsx
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 8
+      name: current_humidity
+      type: integer
+      unit: "%"
+      class: measurement
+      readonly: true
+      range:
+        min: 15
+        max: 100
+    - id: 2
+      name: mode
+      type: string
+      mapping:
+        - dps_val: "dehumidity"
```

Comment:
If there is only one option, it does not seem worth adding it as mode, since the user cannot change it anyway.

Reply:
I think it has to be there, otherwise when exposed to Homekit, it doesn't seem to work as well.  Equally happy to remove if I'm just doing it wrong.